rpy2/situation.py

```diff
@@ -122,9 +127,9 @@ def get_rlib_path(r_home: str, system: str) -> str:
         # i386
         os.environ['PATH'] = ''.join(
```

Comment:
Does this actually work? Usually PATH should be semicolon separated on Windows. On my machine at least joining using the empty string corrups the PATH environment variable and still does not load R properly. I would expect that it should be:
```python
        os.environ['PATH'] = ';'.join(
```

Reply:
Thanks. I did not review the PR as it was failing checks.
This particular issue came up in the context of issue #754. I just fixed it in cross-platform way with revision 9b1fa90bd947be90d6bf8a93aae0e5de0f4d2e90.